Engine/Code/Engine/Window/Window.cpp

```diff
@@ -110,10 +109,17 @@ void Window::Init(const std::string& appName)
 
 	::RegisterClassEx(&wc);
 
+	RECT wr;
+	wr.left = 0;
```

Comment:
could potentially pass these as default arguments to the function and override them in case you want to

Reply:
done

Reply:
done